stripe/src/main/java/com/stripe/android/LoggingUtils.java

```diff
@@ -140,6 +144,34 @@
                 EVENT_DELETE_SOURCE);
     }
 
+    @NonNull
+    static Map<String, Object> getPaymentIntentConfirmationParams(
+            @NonNull Context context,
+            @Nullable List<String> productUsageTokens,
+            @NonNull String publishableApiKey) {
+        return getEventLoggingParams(
+                context,
+                productUsageTokens,
+                null,
```

Comment:
I think you probably want to reach into the `PaymentIntent.source_data` object and pull the `type` out if it's there, because if it is, this is both `source_creation` *and* `confirm`